store/classes.patterns.js

```diff
@@ -94,12 +95,98 @@ const classPatterns = {
 
   [`^shadow-${digitPattern}-${digitPattern}-${digitPattern}-${digitPattern}-${colorPattern}$`]: ".shadow-$1-$2-$3-$4-$5 { box-shadow: $1px $2px $3px $4px $5; }",
 
-
-
-
-
-
-
+  [`^caret-${colorPattern}}`]: "caret-$1 { caret-color: $1; }",
+  [`^clip-${digitPattern}-${digitPattern}-${digitPattern}-${digitPattern}$`]: ".clip-$1-$2-$3-$4 { clip: rect($1px, $2px, $3px, $4px); }",
+  [`^column-count-${digitPattern}$`]: ".column-count-$1 { column-count: $1; }",
+  [`^column-gap-${digitPattern}$`]: ".column-gap-$1 { column-gap: $1px; }",
+  [`^column-rule-${digitPattern}-${lineStylePattern}-${colorPattern}$`]: ".column-rule-$1-$2-$3 { column-rule: $1px $2 $3; }",
+  [`^column-color-${colorPattern}$`]: ".column-color-$1 { column-rule-color: $1; }",
+  [`^column-style-${lineStylePattern}$`]: ".column-style-$1 { column-rule-style: $1; }",
+  [`^column-rule-width-${digitPattern}$`]: ".column-rule-width-$1 { column-rule-width: $1px; }",
+  [`^column-width-${digitPattern}$`]: ".column-width-$1 { column-width: $1px; }",
+  [`^columns-${digitPattern}-${digitPattern}$`]: ".columns-$1-$2 { columns: $1px $2; }",
+  [`^flex-basis-${digitPattern}$`]: ".flex-basis-$1 { flex-basis: $1px; }",
+  [`^flex-grow-${digitPattern}$`]: ".flex-grow-$1 { flex-grow: $1; }",
+  [`^flex-shrink-${digitPattern}$`]: ".flex-shrink-$1 { flex-shrink: $1; }",
+  [`^grid-area-${digitPattern}-${digitPattern}-${digitPattern}-${digitPattern}$`]: ".grid-area-$1-$2-$3-$4 { grid-area: $1 / $2 / span $3 / span $; }",
+  [`^grid-auto-cols-${digitPattern}$`]: ".grid-auto-columns-$1 { grid-auto-columns: $1px; }",
+  [`^grid-auto-rows-${digitPattern}$`]: ".grid-auto-rows-$1 { grid-auto-rows: $1px; }",
+  [`^grid-cols-gap-${digitPattern}$`]: ".grid-cols-gap-$1 { grid-column-gap: $1px; }",
+  [`^grid-gap-${digitPattern}$`]: ".grid-gap-$1 { grid-gap: $1px; }",
+  [`^grid-row-gap-${digitPattern}$`]: ".grid-row-gap-$1 { grid-row-gap: $1px; }",
+  [`^inline-size-${digitPattern}$`]: ".inline-size-$1 { inline-size: $1px; }",
+  [`^inset-${digitPattern}$`]: ".inset-$1 { inset: $1px; }",
+  [`^inset-block-${digitPattern}$`]: ".inset-block-$1 { inset-block: $1px; }",
+  [`^inset-block-${digitPattern}$`]: ".inset-block-$1-$2 { inset-block: $1px $2px; }",
+  [`^inset-block-end${digitPattern}$`]: ".inset-block-end-$1 { inset-block-end: $1px; }",
+  [`^inset-block-start${digitPattern}$`]: ".inset-block-start-$1 { inset-block-start: $1px; }",
+  [`^inset-inline-${digitPattern}$`]: ".inset-inline-$1 { inset-inline: $1px; }",
+  [`^inset-inline-${digitPattern}$`]: ".inset-inline-$1-$2 { inset-inline: $1px $2px; }",
+  [`^inset-inline-end${digitPattern}$`]: ".inset-inline-end-$1 { inset-inline-end: $1px; }",
+  [`^inset-inline-start${digitPattern}$`]: ".inset-inline-start-$1 { inset-inline-start: $1px; }",
+
+  [`^letter-spacing-${digitPattern}$`]: ".letter-spacing-$1 { letter-spacing: $1px; }",
```

Comment:
Let's make a duplicate of this one and call it text-spacing. So that the user can use either rule to achieve the same thing